### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

**Comment by Stanzilla** [comment]
@mrbuds can you do the classic testing for this, please?

**Comment by mrbuds** [comment]
I have tested enable the new extra load conditions for classic and they work fine
Only extra-thing to do is remove talent "known" options by adding bottom of Types.lua
```Lua
  Private.talent_extra_option_types[0] = nil
  Private.talent_extra_option_types[2] = nil
```

**Comment by InfusOnWoW** [comment]
Do they even appear? I though I disabled them entirely for classic?

**Comment by mrbuds** [comment]
> 
> 
> Do they even appear? I though I disabled them entirely for classic?

No they don't, i removed all the `not WeakAuras.IsClassic() and ` + what i said in previous comment to make it work for classic

**Comment by InfusOnWoW** [comment]
So you want it for classic?

**Comment by mrbuds** [comment]
Yes, i have trouble pushing to your repo, here's a diff https://pastebin.com/vCKLjG2D

**Comment by mrbuds** [comment]
diff is wrong way https://pastebin.com/9WPN0xyQ

File: WeakAuras/GenericTrigger.lua [hunk 1]
```diff
@@ -3817,6 +3817,15 @@ WeakAuras.GetItemSubClassInfo = function(i)
   return GetItemSubClassInfo(classId, subClassId)
 end
 
+WeakAuras.GetCritChance = function()
+  -- Based on what the wow paper doll does
+  local spellCrit = GetSpellCritChance(2)
+  for i = 3, MAX_SPELL_SCHOOLS do
```
Comment (mrbuds) [inline WeakAuras/GenericTrigger.lua:3823]: MAX_SPELL_SCHOOLS does not exists on classic

File: WeakAuras/Prototypes.lua [hunk 2]
```diff
@@ -7417,7 +7417,7 @@ Private.event_prototypes = {
         name = "criticalrating",
         display = L["Critical Rating"],
         type = "number",
-        init = "GetCombatRating(CR_CRIT_SPELL)",
+        init = "max(GetCombatRating(CR_CRIT_MELEE), GetCombatRating(CR_CRIT_RANGED), GetCombatRating(CR_CRIT_SPELL))",
```
Comment (mrbuds) [inline WeakAuras/Prototypes.lua:7420]: > print(CR_CRIT_MELEE, CR_CRIT_RANGED, CR_CRIT_SPELL)
nil, nil, nil

Comment (InfusOnWoW) [reply WeakAuras/Prototypes.lua:7420]: But that's all disabled on classic.

Comment (mrbuds) [reply WeakAuras/Prototypes.lua:7420]: alright then :D

File: WeakAuras/GenericTrigger.lua (hunk 1 above)
Comment (InfusOnWoW) [reply WeakAuras/GenericTrigger.lua:3823]: Oh, hrmpf, what does the PaperDollFrame.lua do on classic...

**Review by mrbuds — APPROVED** [review]

**Pull request merged**